code/game/objects/items/weapons/legcuffs.dm

```diff
@@ -151,6 +151,9 @@
 	name = "bola"
 	desc = "A restraining device designed to be thrown at the target. Upon connecting with said target, it will wrap around their legs, making it difficult for them to move quickly."
 	icon_state = "bola"
+	item_state = "bola"
+	lefthand_file = 'icons/mob/inhands/items_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/items_righthand.dmi'
```

Comment:
this is inherited from `/obj/item` so its not needed.

Reply:
Corrected this, should be good now!